### PR TITLE
Twemoji Gem v3.0.0 (twemoji.js V2 support)

### PR DESCRIPTION
**Pull request by JuanitoFatas**
[twemoji.js V2](https://github.com/twitter/twemoji#breaking-changes-in-v2) support 🎉 🎊 ㊗️ 🍻 

- [x] README [Rendered](https://github.com/jollygoodcode/twemoji/blob/twemoji-v3/README.md)
- [x] changelog [Rendered](https://github.com/jollygoodcode/twemoji/blob/twemoji-v3/CHANGELOG.md)
- [x] bump version to 3.0.0
- [x] tests all passed

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@winston @choonkeat Please take a look! 🙇

**Comment by JuanitoFatas** [comment]
a6b7cd8 can be simplified to

```ruby
> /(#{[":+1:"].map { |name| Regexp.quote(name) }.join("|") })/
=> /(:\+1:)/

> /(#{[":+1:"].each { |name| Regexp.quote(name) }.join("|") })/
=> /(:+1:)/
```

File: lib/twemoji.rb [hunk 1]
```diff
@@ -249,7 +218,7 @@ def self.emoji_url(name)
       code = find_by_text(name)
 
       if options[:file_ext] == ".png"
-        File.join(options[:asset_root], options[:image_size], "#{code}.png")
+        File.join(options[:asset_root], "72x72", "#{code}.png")
```
Comment (winston) [inline lib/twemoji.rb]: I don't think "72x72" is used elsewhere, but still it looks like it deserves to be a constant?

Comment (JuanitoFatas) [inline lib/twemoji.rb]: twemoji.js V2 only has one size 72x72 for PNG. Can extract to constant 👍

**Comment by winston** [comment]
👍

**Pull request merged**